index.js

```diff
@@ -1,14 +1,12 @@
 "use strict"
 
-const { default: ow } = require("ow")
-
 const formats = [/https:\/\/drive\.google\.com\/file\/d\/(?<id>.*?)\/(?:edit|view)\?usp=sharing/, /https:\/\/drive\.google\.com\/open\?id=(?<id>.*?)$/]
 
 const alphanumeric = /^[a-zA-Z0-9\-_]+$/
 
 module.exports = (url, apiKey) => {
-	ow(url, ow.string.minLength(1))
-	ow(apiKey, ow.optional.string.minLength(1).matches(alphanumeric))
+	if (!(typeof url === "string") || (typeof url === "string" && !url)) throw new Error("Invalid URL provided.")
```

Comment:
These checks could just be `typeof value !== type`

Reply:
You are right, I was just trying to equivalently replace `ow`.